docs/book/src/component-config-tutorial/api-changes.md

```diff
@@ -67,3 +67,30 @@ mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), options)
 With that out of the way, we can get on to defining our new config!
 
 [tutorial-source]: https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/component-config-tutorial/testdata/project
+
+## If you miss flag `--component-config` during the `kubebuilder init` command, then add it manually as shown below.
```

Comment:
@laxmikantbpandhare see that we have an subsection `Setting up an exising project` where add the steps about what you need to do if you do not scaffold the project with the flag:
https://github.com/kubernetes-sigs/kubebuilder/blob/9dcd389aa9e2386af20b08e045d0334e788c768b/docs/book/src/component-config-tutorial/api-changes.md#setting-up-an-exising-project

So, ihmo it needs to be +1 step under this subsection instead of a new one.

Reply:
Also, we need to add here the step to add to the file: 

```
generatorOptions:
  disableNameSuffixHash: true

configMapGenerator:
- name: manager-config
  files:
  - controller_manager_config.yaml
```

Reply:
Updated file and made above as subsection in `Setting up an existing project`. Also, I added the above missing content.